src/Jobs/ImportSingleProductJob.php

```diff
@@ -117,22 +159,35 @@ private function importVariants(array $variants, string $product_slug)
                     ->collection('variants')
                     ->slug($variant['id']);
             }
+            
+            $data = [
+                'variant_id' => $variant['id'],
+                'product_slug' => $product_slug,
+                'title' => $variant['title'] === 'Default Title' ? 'Default' : $variant['title'],
+                'inventory_quantity' => $variant['inventory_quantity'],
+                'inventory_policy' => $variant['inventory_policy'],
+                'inventory_management' => $variant['inventory_management'],
+                'price' => $variant['price'],
+                'compare_at_price' => $variant['compare_at_price'],
+                'sku' => $variant['sku'],
+                'grams' => $variant['grams'],
+                'requires_shipping' => $variant['requires_shipping'],
+                'option1' => $variant['option1'],
+                'option2' => $variant['option2'],
+                'option3' => $variant['option3'],
+                'storefront_id' => base64_encode($variant['admin_graphql_api_id'])
+            ];
+
+            if ($variant['image_id']) {
+                foreach (($this->data['images'] ?? []) as $image) {
```

Comment:
`collect`?

Reply:
Not sure this makes it cleaner - keeping this one an array is shorter/more readable I think.